src/Planning/understand-options-azure-devops-environments.md

```diff
@@ -25,17 +25,17 @@ While both Azure DevOps Server and Azure DevOps provide the same essential servi
 
 ## Deployment Options
 
-[Azure DevOps Server](https://visualstudio.microsoft.com/tfs) (TFS) and Visual Studio Azure DevOps Server (Azure DevOps ) both provide an integrated collaboration solution. To [understand differences between Azure DevOps and Azure DevOps Server](https://docs.microsoft.com/en-us/azure/devops/user-guide/about-azure-devops-services-tfs?view=azure-devops), read [Essential Services](https://docs.microsoft.com/en-us/vsts/user-guide/services?view=vsts).
+[Azure DevOps Server](https://visualstudio.microsoft.com/tfs) and Visual Studio Azure DevOps Server (Azure DevOps ) both provide an integrated collaboration solution. To [understand differences between Azure DevOps and Azure DevOps Server](https://docs.microsoft.com/en-us/azure/devops/user-guide/about-azure-devops-services-tfs?view=azure-devops), read [Essential Services](https://docs.microsoft.com/en-us/vsts/user-guide/services?view=vsts).
```

Comment:
Change to : https://visualstudio.microsoft.com/team-services/ ?

Reply:
I'll change it to https://azure.microsoft.com/en-us/services/devops/server/